parsons/google/google_cloud_storage.py

```diff
@@ -453,27 +450,33 @@ def copy_bucket_to_gcs(
 
         while polling:
             if latest_operation_name:
+
                 operation = client.get_operation({"name": latest_operation_name})
 
                 if not operation.done:
                     logger.debug("Operation still running...")
 
                 else:
-                    if int(operation.error.code) not in StatusCode["OK"].value:
+                    operation_metadata = storage_transfer.TransferOperation.deserialize(
+                        operation.metadata.value
+                    )
+                    error_output = operation_metadata.error_breakdowns
+                    if len(error_output) != 0:
                         raise Exception(
-                            f"""{blob_storage} to GCS Transfer Job {create_result.name} failed with error: {operation.error.message}
+                            f"""{blob_storage} to GCS Transfer Job {create_result.name} failed with error: {error_output}
                             """
                         )
-                    if operation.response:
+                    else:
                         logger.info(f"TransferJob: {create_result.name} succeeded.")
-                        return
+                    return
 
             else:
                 logger.info("Waiting to kickoff operation...")
                 get_transfer_job_request = storage_transfer.GetTransferJobRequest(
                     {"job_name": create_result.name, "project_id": self.project}
                 )
                 get_result = client.get_transfer_job(request=get_transfer_job_request)
+                logger.info(f"get_result: {get_result}")
```

Comment:
Could this be a debug log, instead of an info? Is this something that would be helpful to someone looking through the log of a higher-level ETL script after it had an error, or something?